packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, Dropdown } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export default function DocumentActions( {
+	templateId,
+	secondaryItem = 'Header',
+	isSecondaryItemActive = false,
+} ) {
+	// TODO: I dislike having this here...
+	const template = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( 'core' );
+
+			return getEntityRecord( 'postType', 'wp_template', templateId );
+		},
+		[ templateId ]
+	);
+
+	return (
+		<div className="edit-site-document-actions">
+			{ template ? (
+				<Dropdown
+					position="bottom center"
+					renderToggle={ ( { onToggle, isOpen } ) => {
+						return (
+							<Button
+								onClick={ onToggle }
+								className="edit-site-document-actions__document_root"
+								aria-haspopup="true"
```

Comment:
I'm unsure which props we need to add to the button

Reply:
Im not certain about a lot of the standard ones, but this seems like a good start.  However, it may also be useful to add a function `onKeyDown` if we want to stay ahead on accessibility:
https://github.com/WordPress/gutenberg/blob/af768dc98c625e33de8023084c6a5c198c52f8ef/packages/block-library/src/heading/heading-level-dropdown.js#L49-L55

Reply:
Nice, thanks for that. Adjusted the props to match that and it's looking much better